### PR TITLE
README: fix typo

### PR DESCRIPTION
**Pull request by moolitayer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Cockpit is perfect for new sysadmins, allowing them to easily perform simple tas
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
 
 #### Multi-server
-You can monitor and administer several servers at the same time. Just add them with a single click and your machines will look after its buddies.
+You can monitor and administer several servers at the same time. Just add them with a single click and your machine will look after its buddies.
```
Comment (moolitayer) [inline README.md]: Not sure if this should be 
```
... and your machine will look after it's buddies.
```
or 
```
... and your machines will look after they're buddies.
```

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Cockpit is perfect for new sysadmins, allowing them to easily perform simple tas
 Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
 
 #### Multi-server
-You can monitor and administer several servers at the same time. Just add them with a single click and your machines will look after its buddies.
+You can monitor and administer several servers at the same time. Just add them with a single click and your machine will look after it's buddies.
```
Comment (moolitayer) [inline README.md]: Not sure if this should be 
```
... and your machine will look after it's buddies.
```
or 
```
... and your machines will look after they're buddies.
```

Comment (petervo) [inline README.md]: I think it should be:

... and your machine will look after its buddies.

Comment (moolitayer) [inline README.md]: right

**Comment by stefwalter** [comment]
Thanks. Merged.

**Pull request closed without merging**